nbs/docs/how-to-guides/MLFlow.ipynb

```diff
@@ -0,0 +1,672 @@
+{
```

Comment:
<p>Please move the import to another cell so that the code doesn't run when generating the docs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>